src/IBI.WZDx/Models/RoadEvents/WorkZoneType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace IBI.WZDx.Models.RoadEvents;
+
+/// <summary>
+/// The type of work zone road event.
+/// </summary>
+public enum WorkZoneType
+{
+    /// <summary>
+    /// The road event is statically placed and is not moving.
+    /// </summary>
+    Static,
+
+    /// <summary>
+    /// The road event is actively moving on the roadway.
+    /// </summary>
+    /// <remarks>
+    /// As opposed to <see cref="PlannedMovingArea"/>, the road event geometry changes at the operation moves. 
+    /// </remarks>
+    Moving,
+
+    /// <summary>
+    /// The planned extent of a moving operation. The active work area will be somewhere within this road event.
```

Comment:
Consider rephrasing the start of the first sentence to be: "The road event is the planned..." for consistency with the Static and Moving definitions.